docs/source/testing/mocking.md

```diff
@@ -84,11 +93,11 @@ const mocks = {
 };
 ```
 
-The previous example uses [casual](https://github.com/boo1ean/casual), a fake data generator for JavaScript, which returns a different result every time the field is called. In other scenarios, such as testing, a collection of fake objects or a generator that always uses a consistent seed are often necessary to provide consistent data.
+The previous example uses [casual](https://github.com/boo1ean/casual), a fake data generator for JavaScript, which returns a different result every time the function is called. In other scenarios, such as testing, a collection of fake objects or a generator that always uses a consistent seed are often necessary to provide consistent data.
 
 ### Using lists in mocks
 
-To automate mocking a list, return an array of the desired length. `[...new Array(n)]` is a convenient syntax for making an array containing *n* copies of `undefined`.
+To automate mocking a list, we recommend returning an array of the desired length. Using `[...new Array(n)]` is convenient syntax for making an array containing *n* copies of `undefined`.
```

Comment:
I think just "return an array" as before is probably better here because I'm not sure there's _another_ way to achieve this

"making an array containing" -> "creating an array that contains" (hadn't thought about this consciously before, but "make" has some ambiguity in the sense of _forcing_, so we should err on the side of "create")